app/app.go

```diff
@@ -133,12 +143,12 @@ var (
 		stakingtypes.NotBondedPoolName: {authtypes.Burner, authtypes.Staking},
 		govtypes.ModuleName:            {authtypes.Burner},
 		ibctransfertypes.ModuleName:    {authtypes.Minter, authtypes.Burner},
-		oracletypes.ModuleName:         nil,
+		oracletypes.ModuleName:         nil, // TODO:??
 	}
 
 	// module accounts that are allowed to receive tokens
 	allowedReceivingModAcc = map[string]bool{
-		oracletypes.ModuleName: true,
+		oracletypes.ModuleName: true, // TODO: why?
```

Comment:
For paying out the people who submit data w/in the reward band.

Reply:
but this allows them to send a transfer to the module address